src/main/java/eu/dissco/core/digitalmediaobjectprocessor/controller/DigitalMediaObjectController.java

```diff
@@ -29,7 +29,6 @@ public class DigitalMediaObjectController {
 
   private final ProcessingService processingService;
 
-  @PreAuthorize("isAuthenticated()")
```

Comment:
This call should be authenticated

Reply:
It is authenticated (See WebSecurityConfig). Anything that isn't a GET request or actuator/health has to be authenticated with a JWT token. This annotation is made redundant